src/wallet/rpcwallet.cpp

```diff
@@ -1725,10 +1728,15 @@ static UniValue gettransaction(const JSONRPCRequest& request)
     uint256 hash(ParseHashV(request.params[0], "txid"));
 
     isminefilter filter = ISMINE_SPENDABLE;
-    if(!request.params[1].isNull())
+    if (!request.params[1].isNull())
```

Comment:
Unrelated change, remove?

Reply:
Thought it would be great to fix this nit while passing through, will remove this change